src/components/Search/SearchOutputGrid.spec.js

```diff
@@ -0,0 +1,14 @@
+import {shallowMount} from "@vue/test-utils";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the latest vue-eslint-parser. See also https://eslint.vuejs.org/user-guide/#what-is-the-use-the-latest-vue-eslint-parser-error](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)